docs/user_guides/fs/feature_view/feature-vectors.md

```diff
@@ -65,7 +90,27 @@ This call will raise an exception if for example for the third entry `pk1 = 5` O
 can't be found.
 That means, `get_feature_vectors` will never return partial feature vector, but will omit empty feature vectors.
 
-If you are aware of missing featurs, you can use the [*passed features*](#passed-features) functionality, described down below.
+If you are aware of missing features, you can use the [*passed features*](#passed-features) or [Partial feature retrieval](#partial-feature-retrieval) functionality, described down below.
+
+### Partial feature retrieval
```

Comment:
Maybe we can point to the fraud_online tutorial to highlight a relevant use case?

Reply:
We don't have an example about imputation yet. The fraud_online tutorial is a use case of passed features.